picard/ui/infodialog.py

```diff
@@ -297,7 +297,7 @@ def _display_info_tab(self):
         if album.errors:
             tabWidget.setTabText(tab_index, _("&Errors"))
             text = '<br />'.join(map(lambda s: '<font color="darkred">%s</font>' %
-                                     '<br />'.join(unicode(QtCore.Qt.escape(s))
+                                     '<br />'.join(unicode(cgi.escape(s))
```

Comment:
I propose to create `picard.util.htmlescape()` so we can change it to whatever fits the best.